src/util.c

```diff
@@ -621,6 +624,19 @@ static int string2llScalar(const char *s, size_t slen, long long *value) {
     return 1;
 }
 
+#if HAVE_IFUNC && HAVE_X86_SIMD
+__attribute__((no_sanitize_address)) static int (*string2ll_resolver(void))(const char *, size_t, long long *) {
+    __builtin_cpu_init();
```

Comment:
Should we add a comment to explain why we need this? Just an idea.

```suggestion
    /* Ifunc resolvers run before ASan initialization and before CPU detection
     * is initialized, so disable ASan and init CPU detection here. */
    __builtin_cpu_init();
```